site/src/components/CourseCard.vue

```diff
@@ -197,12 +197,11 @@ export default class CourseCard extends Vue {
   expanded = this.startExpanded ? this.startExpanded : false;
 
   get credMin(): string {
-    return (
-      this.course.sections[0].credMin +
-      (this.course.sections[0].credMin !== this.course.sections[0].credMax
-        ? "-" + this.course.sections[0].credMax
-        : "")
-    );
+    var min = 2147483647; // arbitrary large number
```

Comment:
There are other ways of finding min/max values that don't need arbitrary integers as well which might be better. 

https://stackoverflow.com/a/36569310

Reply:
Just realized I can use `reduce` for this. I'll be back in a moment.

Reply:
Used reduce with `-Infinity` and `Infinity` instead of integers. This is the best I could think of without having to create an extra list.